src/BrazeMessageComponent.tsx

```diff
@@ -46,6 +48,7 @@ const COMPONENT_MAPPINGS: ComponentMapping = {
     [US_NEWSLETTER_EPIC_NAME]: USNewsletterEpic,
     [AU_NEWSLETTER_EPIC_NAME]: AUNewsletterEpic,
     [UK_NEWSLETTER_EPIC_NAME]: UKNewsletterEpic,
+    Epic: Epic,
```

Comment:
Shall we import the name here from the component file, in the same way we do for the other components?

Reply:
Done